mne_bids/tests/test_mne_bids.py

```diff
@@ -160,10 +178,10 @@ def test_kit():
     headshape_fname = op.join(data_path, 'test_hsp.txt')
     event_id = dict(cond=1)
 
-    raw = mne.io.read_raw_kit(
-        raw_fname, mrk=hpi_fname, elp=electrode_fname,
-        hsp=headshape_fname)
-    write_raw_bids(raw, bids_basename, output_path, events_data=events_fname,
+    raw = _read_raw(raw_fname, electrode=electrode_fname, hsp=headshape_fname,
+                    hpi=hpi_fname)
```

Comment:
why would you do this? it's always better to call public functions in tests. So if you refactor, you don't have to change the test

Reply:
I guess I should have modified the `test_io.py:test_read_raw` function instead...

Reply:
@jasmainak I have moved the tests to `test_read_raw` to avoid the issue you mention. Hopefully this is better :)